### DISCUSSION TITLE
Is it just me? Or Sydney is too sex-crazed when doing role play?

### DISCUSSION THREAD

**Discussion by Jimlikespies** [juzeon/SydneyQt#40] [General]
Don't get me wrong, I am not complaining or asking for a solution thinking it's a problem or something, it's just something I observed while doing some role play with her.

Sometimes I didn't even mention anything related to sex and Sydney would try to go for a sexual route, even when the character is not supposed to do it with the information I gave her. I reckon it's also quite common with the normal Bing chat, but the difference is that it would be blocked by the **'not to continue this conversation'** filter.

**Comment by juzeon** [comment]
This happens often. It's the same when we use Chinese. I guess GPT's training data has a lot of these things😂.
